help_to_heat/frontdoor/views.py

```diff
@@ -161,12 +161,12 @@ def get_prev_next_page_name(page_name, session_id=None):
         is_social_housing = session_data.get("own_property") == "No, I am a social housing tenant"
         receives_benefits = session_data.get("benefits") == "Yes"
 
-    if is_park_home:
-        mapping = schemas.page_prev_next_map_park_home
-        order = schemas.page_order_park_home
-    elif is_social_housing:
+    if is_social_housing:  # This question is asked first, so this path should be checked first to take precedence
```

Comment:
```suggestion
    if is_social_housing:  # This question is asked first, so this path should take precedence in case users have gone back and changed their answers
```

Reply:
Yes, I think that would be clearer